db/queries/assessment_records/queries.py

```diff
@@ -105,3 +105,26 @@ def find_answer_by_key_runner(field_key: str, app_id: str) -> List[tuple]:
         .filter(AssessmentRecord.application_id == app_id)
         .one()
     )
+
+
+def find_assessment(application_id: str):
+    """find_assessment Given an application id `application_id` we return the
+    matching row from the `assessment_records` table.
+
+    :param application_id: The application id of the queried row.
+    :type application_id: str
+    :return: The matching row from the `assessment_records` table.
+    :rtype: AssessmentRecord
+    """
+
+    assessment_record = (
+        db.session.query(AssessmentRecord)
+        .filter(AssessmentRecord.application_id == application_id)
+        .one()
+    )
```

Comment:
⛏️ Change to new SQLA syntax. So, `stmt = select(AssessmentRecord).where(AssessmentRecord.application_id == application_id)` and then execute the stmt using `db.session.scalar(stmt).one()`.

Reply:
Thanks for the review!

This was a good shout, I've added it here: https://github.com/communitiesuk/funding-service-design-assessment-store/pull/45/commits/ff4aa11dbf05cb6f41bf0779f4efccc36d5a3c6d#diff-67a194b656540c4b2f66084a6833fa3d66fbc587d4824be2dc919d8286e5a536R119-R123

I don't think the `.one()` is required since we're using `.scalar(...)`